client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -76,10 +92,10 @@ class SelectBusinessType extends Component {
 						comment: 'In the context of a business activity, brick and mortar or online service',
 					} ) }
 					mainText={ translate(
-						'My business has a physical location customers can visit, ' +
+						'Your business has a physical location customers can visit, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'My business has a physical location customers can visit, or provides goods and services to local customers, or both.' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).